### ISSUE TITLE
[BUG] QSelect with multiple chips, new-value-mode "add-unique" and dropdown menu conflict

### ISSUE THREAD

**Issue by angelhdzmultimedia** [quasarframework/quasar#13600]
### What happened?

I am trying the `new-value-mode="add-unique"` with multiple, use-chips and use-input.

When I select an item from the autocomplete list, it gets added in the QSelect fine.
But when I type to add an item that is not in the autocomplete list, and hit enter key, instead of the item being added in the QSelect, the item in the autocomplete menu gets added instead.

This also causes that if the item that was in the autocomplete list was already added in the QSelect, as I have the `new-value-mode="add-unique"`  it will be removed.

https://user-images.githubusercontent.com/10191085/171882361-9913c584-fc83-4592-ac89-455aa71d1fc0.mp4

This was happened in a Vite JavaScript project locally, with Quasar added as dependency with @quasar/vite-plugin.
Then I reproduced it in a fresh Codepen.

### What did you expect to happen?

I expected being able to add items as chips in the QSelect, either from the autocomplete list, or by typing new items, and preventing repeated items by using `new-value-mode="add-unique"`.

### Reproduction URL

https://codepen.io/angelhdzmultimedia/pen/LYQrmeB

### How to reproduce?

1. Visit https://codepen.io/angelhdzmultimedia/pen/LYQrmeB
2. Click on the QSelect in the QTable's last column.
3. Select one item from the autocomplete list.
4. Now type in the QSelect to add a new item.
5- Instead of the typed item  getting added in the QSelect, the item of the autocomplete that had the focus gets added instead.

Another issue came up:

1. While having `new-value-mode="add-unique"`, add a new item.
2. Add the same item again.
3. Instead of ignoring it, because it already exist, as per the `add-unique` description, it gets deleted and added, behaving like the `toggle` option.

### Flavour

UMD

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by metalsadman** [comment]
it's not a bug, the situation is the option is highlighted, enter key selects it, since it's already selected then it deselects that option. you need to workaround this, like hiding the popup when you are typing (seems fine to me since you are not filtering anyway). https://codepen.io/metalsadman/pen/ZErRwRM

**Comment by metalsadman** [comment]
it's not a bug to me, the situation is the option is highlighted, enter key selects it, since it's already selected then it deselects that option. you need to workaround this, de-highlight the option(pressing up/down key might not be ideal since it's manual), or hiding the popup when you are typing(seems fine to me since you are not filtering anyway). 
https://codepen.io/metalsadman/pen/ZErRwRM

**Comment by angelhdzmultimedia** [comment]
> it's not a bug to me, the situation is the option is highlighted, enter key selects it, since it's already selected then it deselects that option. you need to workaround this, de-highlight the option(pressing up/down key might not be ideal since it's manual), or hiding the popup when you are typing(seems fine to me since you are not filtering anyway). https://codepen.io/metalsadman/pen/ZErRwRM

Ok so the behavior of it confused me.

Im in mobile now, and the enter key gets replaced with a search button, that didn't happen on desktop.

When I typed new items that didn't exist in the autocomplete list, they got added as chips, so I assumed I either could select items, and add new items and they would be added in props.row.roles as it is reactive.


So what I understand now is, this component is not intended for adding new items manually?

And if that's true, why the use-input doesn't prevent of adding items that doesn't exist in the autocomple list?

Do I need a different component to be able to both select existing items, and add new items?

Do I need to use slots to add the "add new item" feature?
